pkg/certificates/certificates.go

```diff
@@ -305,7 +305,24 @@ func (a *AuthorizationDB) UserIDs() ([]string, error) {
 	return keys.Strings(), nil
 }
 
-// Claim marks an authorization as claimed and records claim information
+// List returns all authorizations in the database.
+func (a *AuthorizationDB) List() (auths Authorizations, _ error) {
```

Comment:
It's hard to follow code with short codes when jumping between packages https://github.com/storj/storj/wiki/Style#variable-naming. Maybe `a` -> `authdb`.

Reply:
:+1: